src/components/newReleaseMessage/newReleaseMessage.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { translate } from 'react-i18next';
+import PropTypes from 'prop-types';
+import regex from '../../utils/regex';
+import FlashMessage from '../toolbox/flashMessage/flashMessage';
+import { TertiaryButton } from '../toolbox/buttons/button';
+
+const NewReleaseMessage = ({
+  t,
+  version,
+  releaseNotes,
+  onClick,
+  ...props
+}) => (
+  <FlashMessage shouldShow {...props}>
+    <FlashMessage.Content>
+      <React.Fragment>
+        <strong>{t('Lisk Hub {{version}}', { version })}</strong>
+        {t(' is out. ')}
+        {releaseNotes.match(regex.releaseSummary)[2]}
```

Comment:
IMO it's better to have the component more presentational, by having the logic to extract `releaseSummary` in `src/utils/newRelease.js` and accept only `releaseSummary` prop.